sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -230,6 +243,35 @@ private Tree createParenthesizedExpression(TreeMetaData metaData, KtParenthesize
     Token rightParenthesis = toSlangToken(element.getLastChild());
     return new ParenthesizedExpressionTreeImpl(metaData, expression, leftParenthesis, rightParenthesis);
   }
+  private Tree createReturnTree(TreeMetaData metaData, KtReturnExpression element) {
+    Tree returnExpression = null;
+    if (element.getReturnedExpression() != null) {
+      returnExpression = createElement(element.getReturnedExpression());
+    }
+    return new ReturnTreeImpl(metaData, toSlangToken(element.getReturnKeyword()), returnExpression);
+  }
+
+  private Tree createBreakTree(TreeMetaData metaData, KtBreakExpression ktBreakExpression) {
+    IdentifierTree label = null;
+
+    KtSimpleNameExpression targetLabel = ktBreakExpression.getTargetLabel();
+    if (targetLabel != null && targetLabel.getIdentifier() != null) {
```

Comment:
can we test second condition?

Reply:
there is no obvious way to test when targetLabel.getIdentifier() = null, despite being nullable in the API